packages/native/src/useLinkBuilder.tsx

```diff
@@ -78,5 +81,35 @@ export default function useLinkBuilder() {
     [linking, navigation]
   );
 
-  return buildLink;
+  const buildAction = React.useCallback(
+    (href: string) => {
+      if (!href.startsWith('/')) {
+        throw new Error(`The path must start with '/' (${href}).`);
+      }
+
+      const { options } = linking;
+
+      const state = options?.getStateFromPath
+        ? options.getStateFromPath(href, options.config)
+        : getStateFromPath(href, options?.config);
+
+      if (state) {
+        const action = getActionFromState(state, options?.config);
+
+        if (action !== undefined) {
+          return action;
+        } else {
+          return CommonActions.reset(state);
+        }
```

Comment:
```suggestion
        return action !== undefined ? action : CommonActions.reset(state);
```
This one could be packed into one-liner